scripts/actions/abilities/pets/automaton/barrage_turbine.lua

```diff
@@ -0,0 +1,49 @@
+-----------------------------------
+-- Barrage Turbine
+-- https://www.bg-wiki.com/ffxi/Barrage_Turbine
+-- https://wiki.ffo.jp/html/23698.html
+-----------------------------------
+---@type TAbilityAutomaton
+local abilityObject = {}
+
+abilityObject.onAutomatonAbilityCheck = function(target, automaton, skill)
+    return 0
+end
+
+abilityObject.onAutomatonAbility = function(target, automaton, skill, master, action)
+    automaton:addRecast(xi.recast.ABILITY, skill:getID(), 180)
+
+    -- Apply overload.
+    -- TODO: This is a placeholder that adds zero overload for now.
+    --       For reference, the full maneuver handling is xi.automaton.onUseManeuver.
+    -- local overload = automaton:addBurden(xi.element.WIND - 1, 0)
+
+    local windManeuvers = master:countEffect(xi.effect.WIND_MANEUVER)
+    windManeuvers = utils.clamp(windManeuvers, 0, 3)
+
+    -- Shots per wind maneuver.
+    local shotCount =
```

Comment:
I believe these numbers may be a bit higher than they should be.  Might need to confirm on retail.

Reply:
TP values line up, seems fine to me